src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        int countSupply = 0;
+        int countBuy = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String[] data = value.split(",");
+                if (data[OPERATION].equals("supply")) {
+                    countSupply += Integer.parseInt(data[VALUE]);
+                } else if (data[OPERATION].equals("buy")) {
+                    countBuy += Integer.parseInt(data[VALUE]);
+                }
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file", e);
+        }
+
+        File toFile = new File(toFileName);
```

Comment:
Similarly, ensure that the `toFile` is writable before attempting to write to it. This can prevent runtime errors if the file path is incorrect or if there are permission issues.